Sources/ReleaseServer/ReleaseServer.WebApi/Controllers/ReleaseArtifactController.cs

```diff
@@ -81,6 +128,16 @@ public async Task<ProductVersionListResponseModel> GetVersions([Required] string
             return productVersions.ToProductVersionListResponse();
         }
         
+        /// <summary>
+        /// Retrieves the artifact of the specified product.
+        /// </summary>
+        /// <param name="product"></param>
+        /// <param name="os"></param>
+        /// <param name="architecture"></param>
+        /// <param name="version"></param>
+        /// <response code="200">There exists a product with the specified parameters.</response>
+        /// <response code="500">There exists no product with the specified parameters.</response>
```

Comment:
Same as above: A 500 error code represents a server-side error. 404 (Not Found) would be more appropriate here.

Reply:
Fixed

Reply:
see https://github.com/Traeger-GmbH/release-server/pull/50/commits/89c4e62a1637ac6d990d290b01c6e126d5ef6900